R/dribble.R

```diff
@@ -47,3 +53,27 @@ as.dribble.list <- function(x, ...) {
     class = c("dribble", "tbl_df", "tbl", "data.frame")
   )
 }
+
+## this let's us pull things out of drive_file column that we'd like
+## as a column in the main dribble
+pull_into_dribble <- function(dribble, pull) {
+
+  if (nrow(dribble) == 1) {
```

Comment:
oh oops, that was from a previous iteration of the function I forgot to delete I'll remove it 🙈